app/src/main/kotlin/ConnectFour/ConnectFourGame.kt

```diff
@@ -18,7 +17,7 @@ class ConnectFourGame {
         val moves = mutableListOf<Int>()
 
         println("Playing Connect 4 as \"o\" against Computer as \"x\"")
-        var playerOrder = 0
+        var playerOrder: Int
```

Comment:
Some of these are Intellij's automatic style fixes. I applied some but there turned out to be too many.